e2e/package.json

```diff
@@ -8,6 +8,7 @@
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
     "test": "npm run e2e:all",
+    "headless": "cypress run",  
```

Comment:
This is only headless because the default browser is electron, which defaults to headless.

Maybe we could instead add a new script for each `e2e:*` script: e.g., `e2e:all` plus `e2e:all:headless`

Or we could default them all to headless chrome. I think I only personally use "headed" mode when using `cypress open`, because `cypress run` shuts the browser down right after the test runs.